src/Graphics/Input.elm

```diff
@@ -17,6 +17,7 @@ To learn about text fields, see the
 import Signal
 import Graphics.Element (Element)
 import Native.Graphics.Input
+import Text -- used internally
```

Comment:
Uh-uh - this should be `Native.Text` (with no comment). It will still fix the bug but isn't quite as nice on the source code level.